scripts/support2019/setup_for_2019.ps1

```diff
@@ -1,11 +0,0 @@
-<#
- # Copies files required for Unity 2019 into the project structure.
- #>
-
-$scriptPath = Split-Path $PSCommandPath -Parent
```

Comment:
Deleting this will cause https://github.com/microsoft/MixedRealityToolkit-Unity/blob/a8793f503f09f9c419fe78dd0699dee19fed4c75/pipelines/templates/common-Unity2019.yml#L9 to fail. Do we have a way to trigger this new path from the pipeline?

Reply:
Good question. I can put them back and remove later

Reply:
added back.